canary/package.json

```diff
@@ -7,9 +7,13 @@
     "install": "yarn react:install && yarn vue:install && yarn angular:install",
     "pull": "./environments/pull-environments.sh",
     "script:install": "yarn --cwd script",
-    "setup:next": "yarn script:install && yarn updatePackage",
+    "setup:next": "yarn script:install && yarn updatePackage:next",
+    "setup:next-release": "yarn script:install && yarn updatePackage:next-release",
+    "setup:v3-next": "yarn script:install && yarn updatePackage:v3-next",
     "test": "yarn --cwd e2e test",
-    "updatePackage": "node script/changeCanaryPackageVersion.mjs",
+    "updatePackage:next": "node script/changeCanaryPackageVersion.mjs next",
+    "updatePackage:next-release": "node script/changeCanaryPackageVersion.mjs next-release",
+    "updatePackage:v3-next": "node script/changeCanaryPackageVersion.mjs v3-next",
```

Comment:
in the future, should we just take the parameter in the script instead of having special scripts for each tag?

Reply:
The script does, but one nice thing about having these special scripts is that there is less room for typo then using the script directly

Reply:
That said, open to thoughts! (will be a separate PR though)